meinberlin/apps/cms/migrations/0039_alter_homepage_body.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2022-10-20 11:35
+
+from django.db import migrations
+import wagtail.blocks
+import wagtail.fields
+import wagtail.images.blocks
+
+
+def empty_to_valid_json(apps, schema_editor):
+    HomePage = apps.get_model('meinberlin_cms', 'HomePage')
+
+    for homepage in HomePage.objects.all():
+        if len(homepage.body) == 0:
+            homepage.body = '{}'
+            homepage.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('meinberlin_cms', '0038_streamfield_add_use_json_field'),
+    ]
+
+    operations = [
+        migrations.RunPython(empty_to_valid_json),
```

Comment:
The migration for the `use_json_field` fails if the `body` of the homepage is blank, which it seems to be initially when starting with a fresh db. Once you edit the homepage you have to set all values. This migration sets the body to a valid json string, this will make the migration work.